src/components/dashboard/sidebar/EnterpriseOffersSummaryCard.jsx

```diff
@@ -13,57 +13,66 @@ import {
 } from './data/constants';
 import SidebarCard from './SidebarCard';
 
-const EnterpriseOffersSummaryCard = ({ className, offer, searchCoursesCta }) => (
-  <SidebarCard
-    title={
-      (
-        <div className="d-flex align-items-center justify-content-between">
-          <h3 className="m-0">{ENTERPRISE_OFFER_SUMMARY_CARD_TITLE}</h3>
-          <Badge
-            variant={ENTERPRISE_OFFER_ACTIVE_BADGE_VARIANT}
-            className="ml-2"
-            data-testid="enterprise-offer-status-badge"
-          >
-            {ENTERPRISE_OFFER_ACTIVE_BADGE_LABEL}
-          </Badge>
-        </div>
-      )
-    }
-    cardClassNames={className}
-  >
-    {offer.remainingBalanceForUser && offer.remainingBalanceForUser !== Number.MAX_VALUE
-      ? (
-        <p data-testid="offer-summary-text-detailed">
-          Apply your <b>${offer.remainingBalanceForUser}</b>{' '}
-          balance to enroll into courses.
-        </p>
-      )
-      : (
-        <p data-testid="offer-summary-text">
-          Apply your organization&apos;s learner credit balance to enroll into courses with no out of pocket cost.
-        </p>
-      ) }
+const EnterpriseOffersSummaryCard = ({
+  className, offers, searchCoursesCta,
+}) => {
+  const totalRemainingBalanceForUser = offers.reduce(
```

Comment:
We now want to display the total sum of funds specifically available to the learner, across all offers.